### PR TITLE
Updates to Compose docs and ENV vars

### PR DESCRIPTION
**Pull request by moxiegirl**
- Compose teamhad forgotten some documentation
- Updated ENV for Distribution also
- Forgot one of the readability sections

Signed-off-by: Mary Anthony mary@docker.com
(cherry picked from commit 3a883672417fcb2b3ac0d57d992285849840bfb2)

Conflicts:
    docs/Dockerfile

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @docker/docs-owners @thaJeztah @jamtur01 @fredlf @SvenDowideit

**Comment by jamtur01** [comment]
LGTM

File: docs/Dockerfile [hunk 1]
```diff
@@ -64,45 +65,72 @@ RUN sed -i.old '1s;^;no_version_dropdown: true;' /docs/sources/registry/spec/jso
...
 RUN sed -i.old '1s;^;no_version_dropdown: true;' /docs/sources/compose/wordpress.md
 
+ADD https://raw.githubusercontent.com/docker/compose/${COMPOSE_BRANCH}/docs/extends.md /docs/sources/compose/extends.md
+RUN sed -i.old '1s;^;no_version_dropdown: true;' /docs/sources/compose/extends.md
+
+ADD https://raw.githubusercontent.com/docker/compose/${COMPOSE_BRANCH}/docs/production.md /docs/sources/compose/production.md
+RUN sed -i.old '1s;^;no_version_dropdown: true;' /docs/sources/compose/production.md
+
 # Then build everything together, ready for mkdocs
 RUN /docs/build.sh
```
Comment (thaJeztah) [inline docs/Dockerfile]: Not introduced in this PR; but "missing newline at end of file"

**Comment by thaJeztah** [comment]
LGTM

File: docs/Dockerfile (hunk 1 above)
Comment (thaJeztah) [inline docs/Dockerfile]: ^^ just a mention, In case you have to do some follow-up work on this file

**Comment by moxiegirl** [comment]
Thanks gents!

**Pull request merged**